pkg/config/service.go

```diff
@@ -171,6 +172,9 @@ func (c *ServiceConfig) InitDefaults() {
 	if c.Latency.RTPMaxAllowedTsDiff == 0 {
 		c.Latency.RTPMaxAllowedTsDiff = defaultRTPMaxAllowedTsDiff
 	}
+	if c.Latency.OldPacketThreshold == 0 {
```

Comment:
Documentation indicates that 0 disables drops. Is it the case here?

Reply:
There are two settings now @biglittlebigben . The current mode of operation gets two extra settings. One to enable use of receive time of packets. If that is enabled, this has an effect. If not, this does not.

For the new mode using one-way-delay estimation, it always uses arrival time of packet. Wanted to enable this by default in the new mode. So, setting default here if not set. The old mode won't be affected unless arrival time is used.

More details in this PR - https://github.com/livekit/server-sdk-go/pull/741